files/chef-marketplace-cookbooks/chef-marketplace/templates/default/nginx-chef-internal.conf.erb

```diff
@@ -0,0 +1,7 @@
+#
+# chef server endpoints proxied by Delivery's nginx
```

Comment:
`delivery['chef_server_proxy'] = true` should create the upstreams for the chef server for you.